scripts/components/icon-label/icon-label.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Icon } from '@wordpress/components';
+
+/**
+ * A simple icon-label combo for streamlined components.
+ * 
+ * @param {React.Component} props.icon       - Icon to display.
+ * @param {React.Component} props.icon       - Icon to display.
```

Comment:
double doc, I assume it should be
```suggestion
 * @param {React.Component} props.label      - Label to display.
 * @param {React.Component} props.icon       - Icon to display.
```